MekHQ/src/mekhq/gui/dialog/CampaignOptionsDialog.java

```diff
@@ -1585,12 +1585,12 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
 
         chkUseTransfers = new JCheckBox("Log Saver - Use Reassign instead of Remove/Assign"); // NOI18N
         chkUseTransfers.setSelected(options.useTransfers());
-        gridBagConstraints.gridy++;
+        gridBagConstraints.gridy = 22;
         panPersonnel.add(chkUseTransfers, gridBagConstraints);
 
         chkUseTimeInService = new JCheckBox("Track Time In Service"); // NOI18N
         chkUseTimeInService.setSelected(options.getUseTimeInService());
-        gridBagConstraints.gridy = 22;
+        gridBagConstraints.gridy = 23;
```

Comment:
Why the reversal to fixed numbers instead of incrementing the `gridy`? Shouldn't it just be incremented for every option in the column instead, so basically changing ln 1593 to ++ and the same any other controls on the page that need it? That would make it slightly easier to modify those options in the future without breaking one.

Reply:
True, and I also have a rather old branch to do just that. Just need to get around to finishing it... I could ask the same of mkerensky who only changed ONE to use ++

Reply:
The issue I know about is the set edge reroll menu, which is disappearing for non-combat personnel after being set once. 

Why I only made a change to one section to gridy++ ?    I either ran into a weird issue when I tried changing more than that, or I was just concentrating on the block I was working on. I'm still learning, here, so I try to leave anything that I'm not directly working on alone.

Reply:
Oh whoops. I guess I misread "nick" instead of your name? It was Nick I was referring to, sorry.

Reply:
For now this fix should stand as is. Really the branch I started is really old and needs work. Also needs to be finished.